app/uk/gov/hmrc/cardpaymentfrontend/views/TimedOutPage.scala.html

```diff
@@ -0,0 +1,42 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
```

Comment:
We should check the copyright headers, where we've copy pasted we've got the one from 2024